src/components/Button/Button.test.js

```diff
@@ -0,0 +1,43 @@
+// Button specific implementation
+// Resources https://stackoverflow.com/questions/58408178/query-a-button-with-specific-text
+
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { Button } from './Button';
+
+describe('Button works as expected', () => {
+    it('Button displays the text provided', async () => {
```

Comment:
There's no need to use _async_ here, this component doesn't have any asynchronous capabilities neither you are using any async code inside the test